.github/workflows/e2e_test_ci.yml

```diff
@@ -272,6 +272,89 @@ jobs:
           killall spice
           cat spice.log
 
+  test_quickstart_graphql:
+      name: "Test GraphQL quickstart (${{ matrix.target.target_os }}-${{ matrix.target.target_arch }})"
+      runs-on: ${{ matrix.target.runner }}
+
+      if: github.event_name == 'workflow_dispatch' || github.event_name == 'push'
+      needs:
+        - build
+        - setup-matrix
+
+      strategy:
+        matrix:
+          target: ${{ fromJson(needs.setup-matrix.outputs.matrix) }}
+
+      steps:
+        - uses: actions/checkout@v4
+
+        - name: download artifacts - build_${{ matrix.target.target_os }}_${{ matrix.target.target_arch }}
+          uses: actions/download-artifact@v4
+          with:
+            name: build_${{ matrix.target.target_os }}_${{ matrix.target.target_arch }}
+            path: ./build
+
+        - name: Install spice
+          uses: ./.github/actions/install-spice
+          with:
+            build-path: ./build
+
+        - name: Init spice app
+          run: |
+            spice init test_app
+
+        - name: Spice dataset configure
```

Comment:
@gloomweaver Kirill please verify this step:
https://github.com/spiceai/spiceai/actions/runs/9323632524/job/25667579082

>Run echo -e "test_graphql_table\nrepos\graphql:[https://api.github.com/graphql\ny](https://api.github.com/graphql/ny)" | spice dataset configure
dataset name: (test_app) description: from: locally accelerate (y/n)? (y) EOF
Error: Process completed with exit code 1.

Reply:
I think it is due to missing 'n' here: `\nrepos\graphql `
Should be
>echo -e "test_graphql_table\nrepos\ngraphql:https://api.github.com/graphql\ny" | spice dataset configure